docs/Serverless.md

```diff
@@ -190,3 +191,73 @@ gcloud beta run deploy --image gcr.io/PROJECT-ID/APP-NAME --platform managed
 ```
 
 Your app will be accessible from the URL GCP provides.
+
+## Zeit Now
+
+[now](https://zeit.co/home) provides zero configuration deployment for
+Node.js applications. In order to use now, it is as simple as
+configuring your `now.json` file like the following:
+
+```json
+{
+  "builds": [
+    {
+      "src": "serverless.js",
+      "use": "@now/node",
+      "config": {
+        "helpers": false
+      }
+    }
+  ],
+  "routes": [
+    { "src": "/.*", "dest": "/serverless.js"}
+  ]
+}
+```
+
+Then, write a `serverless.js` like so:
+
+```js
+'use strict'
+
+const build = require('./index')
+
+const app = build()
+
+module.exports = async function (req, res) {
+  await app.ready()
+  app.server.emit('request', req, res)
+}
+```
+
+And a `index.js` file:
```

Comment:
This file should be "moved" to the directory `/api/index.js`

Reply:
It doesn't. The `build` section does not include `/api`. I'm happy to add it to both if you prefer.

Reply:
I didn't write all the info:
you have right, the example works, but if a user runs `now dev` it doesn't work without that structure so, I have afraid we could have some issue about that.